.github/workflows/build.yml

```diff
@@ -36,13 +36,13 @@ jobs:
       run: nox -s test-${{ matrix.python-version }}
 
     - name: Run coverage test
-      if: matrix.python-version == 3.8
+      if: matrix.python-version == 3.7
```

Comment:
not sure if this was intentional, it used to test the latest version and now this will be testing with the oldest version

Reply:
Yes, I think the version we use for coverage is not relevant and int this way we don't have to update it here so frequently.